src/MOI_wrapper.jl

```diff
@@ -231,28 +251,61 @@ function MOI.optimize!(instance::Optimizer)
     b = instance.data.b
     G = ECOS.ECOSMatrix(sparse(instance.data.IG, instance.data.JG, instance.data.VG, m, n))
     h = instance.data.h
-    objconstant = instance.data.objconstant
+    objective_constant = instance.data.objective_constant
     c = instance.data.c
     instance.data = nothing # Allows GC to free instance.data before A is loaded to ECOS
+    options = instance.options
+    if instance.silent
+        options = copy(options)
+        options[:verbose] = false
+    end
     ecos_prob_ptr = ECOS.setup(n, m, cone.f, cone.l, length(cone.qa), cone.qa,
-                               cone.ep, G, A, c, h, b; instance.options...)
+                               cone.ep, G, A, c, h, b; options...)
+    start_time = time()
```

Comment:
Nit: If setup is required for the solve, shouldn't it be included in the solve time?

Reply:
Makes sense, setup time is kind of solver specific